routers/router.dal1.yml

```
@@ -1,4 +1,16 @@
 ---
+- name: BSDROCKER
+  asn: 4242421022
+  ipv6: fd2e:5e5e:83ad::1022:207
+  multiprotocol: true
+  extended_nexthop: true
```

```suggestion
  ipv4: <FIXME: add an IPv4 address for this tunnel>
  ipv6: fd2e:5e5e:83ad::1022:207
  multiprotocol: true
```

Let's try this without enabling `extended_nexthop` given the difficulties in #216.